src/paperless/views.py

```diff
@@ -166,4 +166,4 @@ class ApplicationConfigurationViewSet(ModelViewSet):
     queryset = ApplicationConfiguration.objects
 
     serializer_class = ApplicationConfigurationSerializer
-    permission_classes = (IsAuthenticated,)
```

Comment:
Well, the configuration object has no owner, so this doesn't do much right?  Because `has_object_permission` will simply return True  As I recall, it was made globally editable deliberately, in much the same way the workflows currently are

Reply:
This is a wonderful point, my bad. We want just guardian permissions, lemme check.

Reply:
And what would it mean for the configuration to have an owner anyway?  The parser doesn't have any knowledge of who triggered the parsing, so there can't (yet) be any idea of per-user configuration.

Reply:
Yea not owner permissions just the guardian object perms

Reply:
Ok I think this should do it, uses the base Django object perms